### PR TITLE
Testing To Do list: part 1✔️ 

### PR DESCRIPTION
**Pull request by NATASHA-ct**
- [ ] Used jest framework for testing. 
- [ ] Created a test file ([..].test. js) for a file containing the add item and delete item functions that we wanted to test.
- [ ] Checked if the functions were pure ?
- [ ] Wrote tests for them 
- [ ] Mocked a storage object to "imitate" localStorage operations (only if you are using jest < 24.x, in newer versions localStorage is mocked automatically).
- [ ] Mocked HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
- [ ] Made sure to group our tests using the describe() method.

### CODE REVIEW

File: src/removeFromLocalStorage.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  const localstorage = JSON.parse(localStorage.getItem('list')) || [];
+  localstorage.push(list);
+  expect(localstorage).toHaveLength(1);
+  localstorage.push(list2);
+  expect(localstorage).toHaveLength(2);
+  localstorage.pop();
+  expect(localstorage).toHaveLength(1);
+});
```
Comment (FrederickMih) [inline src/removeFromLocalStorage.test.js]: - [x] Kindly make sure to test the function `removeDataFromLocalStorage()`. You commented the importation meaning that you are no longer testing the function. 
- [x] You'll require to add tests for most pure functions
- [x] Tests should be group using the description() method

File: src/removeFromLocalStorage.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// const removeDataFromLocalStorage = require('./removeFromLocalStorage.js');
```
Comment (FrederickMih) [inline src/removeFromLocalStorage.test.js]: - [x] Comments make your codebase look messy. If you need a previous version of your code, use git history.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




# To Highlights

- PR has a title and summary :heavy_check_mark: 
- Linter checks are Ok :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/removeFromLocalStorage.test.js (hunk 1 above)
Comment (aneallaryea100) [reply src/removeFromLocalStorage.test.js]: * function , removeDataFromLocalStorage() imported.
* Tests group using description method.
* Pure functions tested.

**Review by Gambit142 — APPROVED** [review]
Hi @aneallaryea100  @NATASHA-ct 👋, 

## Status: Approved 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great work implementing the changes suggested by the previous reviewer. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**